exotica_core/include/exotica_core/kinematic_element.h

```diff
@@ -121,6 +121,8 @@ class KinematicElement
     KDL::Frame generated_offset = KDL::Frame::Identity();
     bool is_trajectory_generated = false;
     std::vector<double> joint_limits;
+    double velocity_limit;
```

Comment:
One more nit: Can you assign a `std::nan` as the default value here?

Reply:
Added to both velocity limit and acceleration limit